tests/models/test_action_predictor.py

```diff
@@ -0,0 +1,19 @@
+import torch
+
+from ami.models.action_predictor import ContinousActionPredictor
+from ami.models.components.fully_connected_fixed_std_normal import DeterministicNormal
+
+
+def test_continous_action_predictor_forward():
+    dim_embed = 16
+    dim_action = 4
+    dim_hidden = 32
```

Comment:
テストに必要な定数は`@pytest.mark.parametrize`で定義したい気がした
こう実装しておけば今後テストケースを増やしたくなった時、例えば`dim_embed`が`1024`の時もテストしたくなったら`@pytest.mark.parametrize("dim_embed", [16])`を`@pytest.mark.parametrize("dim_embed", [16, 1024])`に変えるだけで`(dim_embed, dim_action, dim_hidden)`が`(16, 4, 32)`の場合と`(1024, 4, 32)`の場合をテストできるようになって楽
```suggestion
@pytest.mark.parametrize("dim_embed", [16])
@pytest.mark.parametrize("dim_action", [4])
@pytest.mark.parametrize("dim_hidden", [32])
def test_continous_action_predictor_forward(
    dim_embed: int, 
    dim_action: int, 
    dim_hidden: int,
):
```

Reply:
修正しました。https://github.com/MLShukai/ami-vconf24/pull/13/commits/c1ad562e822bd397bd5ceb976e2e1f1c7bbefb9d